### ISSUE TITLE
Tweet Sum Discrepancy

### ISSUE THREAD

**Issue by nbuhagiar** [hashkat/hashkat#48]
Running a network simulation, I found in the 'main_stats'  output file that there were a total number of 5104 tweets in this simulation. However, looking into the 'Categories_Distro.dat' file, where I set the tweet bin spacing to 1 and calculated the total number of tweets using the information given in this file, I found the value for the total number of tweets to be 4168. Some of the data in the 'Cateories_Distro.dat' file is shown below:

![categories_distro_tweets](https://cloud.githubusercontent.com/assets/10453145/8734929/5592660a-2bdb-11e5-8a2f-d4388693cce2.png)

Do you have any idea what could be the possible cause for this discrepancy?

**Comment by kryczko** [comment]
Were there any retweets that occurred?

**Comment by nbuhagiar** [comment]
Yes, I've attached the 'main_stats' file to this message if that helps.

**Comment by kryczko** [comment]
I don't see the file. Could you reattach it?

**Comment by nbuhagiar** [comment]
You might not be able to see attached files to messages on github so I've
attached the file to an email I sent to you.

Thanks for your help!
